README.md

```diff
@@ -60,6 +61,7 @@ The real-life examples that uses the code *isogram* generates in their pages
 | ----------------------------- | ---------------------------- | --------------------------- |
 | apiDoc                        | [apidocjs.com][apidoc]       | `a` `p` `i` `d` `o` `c`     |
 | BrowserSync                   | [browsersync.io][bsync]      | `s` `y` `n` `c` `I` `t`     |
+| Choose a License              | [choosealicense.com][license]| `L` `I` `C` `e` `N` `S` `E` |
```

Comment:
wouldn't that misalign the table's separators? Perhaps an extra space could be added to all cells in the second column to compensate.